src/astgen

```diff
@@ -712,6 +733,8 @@ Args = parser.parse_args()
 
 read_types(Args.I + "/V3Ast.h")
 read_types(Args.I + "/V3AstNodes.h")
+read_types(Args.I + "/V3AstNodeDType.h")
+read_types(Args.I + "/V3AstNodeMath.h")
```

Comment:
Pass down as args from the Makefile instead so we make sure that the makefile's rules are also consistent?